src/data/loadToDb.ts

```diff
@@ -0,0 +1,44 @@
+//import AWS from 'aws-sdk';
+import { productsDB } from './productsForLoad';
+import { stockDB } from './stockForLoad';
+import { IProduct, IStock, StatusCode } from '../types/types';
+import { generateHttpResponse } from '../utils/lambda';
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient, PutCommand } from "@aws-sdk/lib-dynamodb";
+
+const dynClient = DynamoDBDocumentClient.from(
```

Comment:
Lets rename to `dynamoDBClient` or `dbClient` - it's more readable

Reply:
Done